### PR TITLE
Python release notes v9.9.0

### PR DESCRIPTION
**Pull request by lrafeei**
This PR adds the release notes for the Python Agent v9.9.0 as well as a new configuration option

### CODE REVIEW

File: src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx [hunk 1]
```diff
@@ -3732,6 +3732,67 @@ Event loop visibility surfaces information about transactions that block the eve
...
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment-variables)
+          </th>
+
+          <td>
+            `NEW_RELIC_MEMORY_RUNTIME_METRICS_ENABLED`
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx:3786]:
```suggestion
            `NEW_RELIC_MEMORY_RUNTIME_PID_METRICS_ENABLED`
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+subject: Python agent
+releaseDate: '2024-04-18'
+version: 9.8.0
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx:4]:
```suggestion
version: 9.9.0
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Update Celery instrumentation', 'Add new Langchain vectorstores', 'Add configuration to capture memory usage runtime metrics per process']
+bugs: ['Report only raw content dictionary for anthropic claude']
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx:7]:
```suggestion
bugs: ['Report only raw content dictionary for Anthropic Claude']
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+security: ['Update internal version of urllib3 to v1.26.18']
+---
+
+## Notes
+
+This release of the Python agent adds
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx:13]:
```suggestion
This release of the Python agent updates Celery instrumentation, adds new Langchain vectorstores, adds configuration to capture memory usage runtime metrics per process, fixes a content reporting issue in Anthropic Claude, and upgrades internal version of urllib3 to v1.26.18.
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+subject: Python agent
+releaseDate: '2024-04-18'
+version: 9.8.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Update Celery instrumentation', 'Add new Langchain vectorstores', 'Add configuration to capture memory usage runtime metrics per process']
+bugs: ['Report only raw content dictionary for anthropic claude']
+security: ['Update internal version of urllib3 to v1.26.18']
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx:8]:
```suggestion
security: ['Upgrade internal version of urllib3 to v1.26.18']
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Notes
+
+This release of the Python agent adds 
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## Security
+
+* **Update internal version of urllib3 to v1.26.18**
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx:19]:
```suggestion
* **Upgrade internal version of urllib3 to v1.26.18**
```

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+    * Support for the following Langchain vectorstores: `DuckDB`, `EcloudESVectorStore`, `InMemoryVectorStore`, `PathwayVectorClient`, `VDMS`
+
+* **Add configuration setting to enable capturing of memory usage runtime metrics per process**
+
+    * Add `memory_runtime_pid_metrics.enabled` configuration setting to toggle capturing of memory usage metrics (per process ID only) to reduce Metric Grouping Issues (MGI).
+
+## Bug fixes
+
+* **Report only raw content dictionary for anthropic claude**
```
Comment (hmstepanek) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90900.mdx:44]:
```suggestion
* **Report only raw content dictionary for Anthropic Claude**
```

File: src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx [hunk 8]
```diff
@@ -3732,6 +3732,67 @@ Event loop visibility surfaces information about transactions that block the eve
...
+          </th>
+
+          <td>
+            `NEW_RELIC_MEMORY_RUNTIME_PID_METRICS_ENABLED`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    When enabled, the agent will generate and send runtime metrics.
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx:3792]:
```suggestion
    When enabled, the agent will generate and send runtime metrics per process ID.
```

**Comment by rhetoric101** [comment]
I am doing an override since this built successfully in Netlify Cloud.

**Pull request merged**